src/modules/fancyzones/lib/FancyZonesData.cpp

```diff
@@ -60,6 +59,84 @@ namespace
 
         return tmpDirPath;
     }
+
+    bool DeleteRegistryKey(HKEY hKeyRoot, LPTSTR lpSubKey)
```

Comment:
Let's make this as simple as possible: try to delete the `SuperFancyZones` key and if it fails we don't care.